src/components/NavBar/Search/Search.component.jsx

```diff
@@ -1,18 +1,55 @@
 import SearchIcon from '@material-ui/icons/Search';
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import CancelIcon from '@material-ui/icons/Cancel';
+import { IconButton } from '@material-ui/core';
 import { StyledInput, IconContainer, SearchContainer } from './Search.styled';
 
-const Search = ({ value }) => {
+const Search = ({ handleSearchChange }) => {
+  const history = useHistory();
+  const [keyWord, setKeyWord] = useState('');
```

Comment:
I don't think we have another state here, we already had `search` state in App component. So we should pass it down here

Reply:
I do not really understand this part

Reply:
yeah maybe just keep it for now. We will fix it after learning state management part